### PR TITLE
Feature/expand single print options

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

File: src/features/courses/course-controller.ts [hunk 1]
```diff
@@ -4505,24 +4543,33 @@ You can contact your student at ${options.student.email} or by replying to this
                             as: 'studentGradeInstance',
                             attributes: ['id', 'webworkQuestionPath'],
                             where: {
-                                active: true,
+                                active: false,
```
Comment (red-tt-bear) [inline src/features/courses/course-controller.ts:4546]: Why do we want active false?

**Review by red-tt-bear — CHANGES_REQUESTED** [review]
active false

File: src/features/courses/course-controller.ts (hunk 1 above)
Comment (PanopticaRising) [reply src/features/courses/course-controller.ts:4546]: Good catch, this was meant to be `required: false`.

File: src/features/courses/course-controller.ts [hunk 2]
```diff
@@ -4512,17 +4550,27 @@ You can contact your student at ${options.student.email} or by replying to this
...
                                     }
                                 }
                             ]
                         },
                     ]
-                });
+                }) : null;
+                
+                let attachments = gradeInstanceAttachments?.studentGradeInstance?.problemAttachments;                
+                if (mainData.topicTypeId === 1) {
```
Comment (red-tt-bear) [inline src/features/courses/course-controller.ts:4564]: I'm surprised you don't use a ternary here

**Review by red-tt-bear — APPROVED** [review]

**Pull request merged**